guides/micronaut-cloud-database-azure/micronaut-cloud-database-azure.adoc

```diff
@@ -0,0 +1,197 @@
+external:micronaut-cloud-database-base/start.adoc[]
+
+We will then connect the application to https://azure.microsoft.com/en-us/services/mysql/[Azure Database for MySQL].
+
+common:azure-cli.adoc[]
+common:azure-account-paid.adoc[]
+
+NOTE: Your Azure account must be a paid account or trial with credits available.
+
+== Create an Azure Database for MySQL instance
+
+The first step in this guide is to create a new https://azure.microsoft.com/en-us/services/mysql/[Azure Database for MySQL] instance. 3 steps are required to achieve that using the https://docs.microsoft.com/en-us/cli/azure/install-azure-cli[Azure CLI]:
+
+1. First we need to create a resource group that the database will form part of. It is recommended to create a new resource group just for this guide and not re-use an existing one.
```

Comment:
I'd remove "need to" in each of these, e.g. "First, create a ...", "Then, create the Azure ..."